dev-docs/bidders/smartytech.md

```diff
@@ -4,7 +4,7 @@ title: SmartyTech
 description: Prebid SmartyTech Bidder Adaptor
 pbjs: true
 biddercode: smartytech
-media_types: banner
+media_types: banner, video
```

Comment:
If you support multi format, I recommend adding `multiformat_supported` (will-bid-on-any, will-bid-on-one, will-not-bid). This helps publishers and sales houses to adjust their ad setups

Reply:
Done